speakers/src/main/java/in/droidcon/speakers/remote/SpeakerMapper.kt

```diff
@@ -0,0 +1,28 @@
+package `in`.droidcon.speakers.remote
+
+import `in`.droidcon.base.repo.remote.EntityMapper
+import `in`.droidcon.speakers.model.SpeakerEntity
+
+object SpeakerMapper : EntityMapper<SpeakerModel, SpeakerEntity> {
+    override fun mapFromRemote(type: SpeakerModel): SpeakerEntity {
+        return SpeakerEntity(
+            speakerId = type.id,
+            speakerName = type.fullName,
+
+            /*Sessionize doesn't provide company name
+             *However, it has an optional field for Company Website
```

Comment:
I know it is a long shot but, we are collecting company name as a part of the talk. So if it would be possible, we can get the company name from the talk they submitted.

Reply:
Hey,
I know company name is collected as part of the talk, but the issue here is that they will share that inside the session object, not inside the speaker object, thus we will need to call `/all` or `/sessions` for that, which I believe wouldn't be a good idea.
Moreover they share those answers in a weird format like below, which makes it even more difficult to use.
```
"questionAnswers": [
  {
    "id": 106,
    "question": "Where do you work?",
    "questionType": "Short_Text",
    "answer": "Byju's",
    "sort": 5,
    "answerExtra": null
  }
]
```

Reply:
Ah yeah I guessed so. That's why called it a long shot 😄 No issues with the current one but wanted to know if the other thing is feasible. Thus the comment :)

Reply:
Got it. We can try something once schedule, sessions, and favorites modules are done.